servers/src/main/java/tachyon/master/file/journal/InodeFileEntry.java

```diff
@@ -103,7 +135,12 @@ public boolean isCacheable() {
   }
 
   @JsonGetter
-  public long getTtl() {
+  public long getTTL() {
     return mTTL;
   }
+
+  @Override
+  public JournalEntryType getType() {
```

Comment:
If you're sorting the methods, could you also sort `getCacheable` and `getCompleted`?